### PR TITLE
feature: Add a new <Text> component. (Do not merge yet)

### PR DESCRIPTION
**Pull request by bytaesu**
<aside>

image_1

image_2

</aside>

# **What is this.**

image_1111

This component simplifies writing text using shadcn-ui and TailwindCSS.

Although there is an existing Typography example, it makes the code look messy due to its length when used. 
So, I created a Text component to manage text throughout the project.

# **For more information.**

I’ve uploaded the component to a public repository. Feel free to check it out!
### [-> Repository](https://github.com/bytaesu/shadcn-ui-text)

# **In conclusion.**

Additional work is still needed, such as a user guide, installation, etc. Once merged, I will work on it further to fit shadcn-ui and submit an additional pull request so that it can be used right away.

### CODE REVIEW

**Comment by bytaesu** [comment]
The contribution guide mentions that when requesting to create a new component, a Discussion should be opened, so I submitted the PR based on that.

I’ll create an issue first, discuss it, and then submit the PR again later!
